src/components/productdisplayitem.main.jsx

```diff
@@ -577,6 +583,36 @@ class ProductDisplayItemMain extends React.Component {
                 ) : ('')
                 }
               </div>
+              <div className="social-network-sharing">
+                <InlineShareButtons
+                  config={{
+                    alignment: 'center', // alignment of buttons (left, center, right)
+                    color: 'social', // set the color of buttons (social, white)
+                    enabled: true, // show/hide buttons (true, false)
+                    font_size: 16, // font size for the buttons
+                    labels: 'cta', // button labels (cta, counts, null)
+                    language: 'en', // which language to use (see LANGUAGES)
+                    networks: [ // which networks to include (see SHARING NETWORKS)
+                      'facebook',
+                      'twitter',
+                      'pinterest',
+                      'email',
+                    ],
+                    padding: 12, // padding within buttons (INTEGER)
+                    radius: 4, // the corner radius on each button (INTEGER)
+                    size: 40, // the size of each button (INTEGER)
+
+                    // OPTIONAL PARAMETERS
+                    url: productLink, // (defaults to current url)
+                    image: shareImage, // (defaults to og:image or twitter:image)
+                    description: 'custom text', // (defaults to og:description or twitter:description)
```

Comment:
Is it possible to pass the product description directly here?
Available in `productData._definition[0].details`